libuserhelper.cpp

```diff
@@ -307,8 +318,19 @@ bool LibUserHelper::modifyUser(uint uid, const QString &newName)
     struct lu_ent *ent = lu_ent_new();
 
     if (lu_user_lookup_id(context, uid, ent, &error)) {
-        QString fixedName(newName);
-        lu_ent_set_string(ent, LU_GECOS, fixedName.remove(',').remove(':').toUtf8());
+        QString realName(newName);
```

Comment:
In the other place similar variable is called nameUuid, I think it would be good to use the same name in both places. It's not totally necessary though but would be easier to see the connection in future.

Reply:
Ok np